app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -3,14 +3,46 @@
   var dom = window.shared.ReactHelpers.dom;
   var createEl = window.shared.ReactHelpers.createEl;
   var merge = window.shared.ReactHelpers.merge;
-
+  var FeedHelpers = window.shared.FeedHelpers;
   var QuadConverter = window.shared.QuadConverter;
 
+  var Colors = {
+    math: '#e8e9fc',
+    reading: '#ffe7d6',
+    procedural: '#e8fce8'
+  };
+  var styles = {
+    feedCard: {
+      marginBottom: '10px',
+      borderRadius: '10px',
+      fontSize: '10pt'
+    },
+    feedCardHeader: {
+      fontSize: '120%',
```

Comment:
Since absences/tardies have `fontSize: 80%` on the whole div, if I didn't use percentages here I would have to make separate styles for `feedCardHeaderCompressed`, `feedCardHeader`, `feedCardBodyCompressed`, etc.